setup/ubuntu.md

```diff
@@ -5,9 +5,9 @@ This document contains steps for installing the [interlibr](https://github.com/X
 ## Installation
 
 How to configure the [interlibr](https://github.com/Xalgorithms/interlibr) compiler and interpreter on Ubuntu 18.:
-1. Clone repositories.
-2. Install dependencies.
-3. Test and contribute.
+[1. Clone repositories.](https://github.com/Xalgorithms/general-documentation/blob/master/setup/ubuntu.md#clone-repositories)
```

Comment:
Heading links must be changed from `[1. topic.](https://heading-link)` to `1. [topic](http://heading-link` to correctly display as a list.

Reply:
Thanks for catching that. Fixed.